Jenkinsfile

```diff
@@ -1,209 +1,211 @@
                                 }
                             }
+                            catch(e) {
+                                build_ok = false
+                                echo e.toString()
+                            }
+                            sh 'source deactivate'
+
+                            try{
+                                env.STAGE_STATUS = "Running app tests on ${label}"
+                                sh """
+                                    source activate ${condaPath}/envs/isis
+                                    echo $PATH
+                                    ctest -R _app_ -j4 -VV
+                                """
+                            }
+                            catch(e) {
+                                build_ok = false
+                                errors.add(env.STAGE_STATUS)
+                                println e.toString()
+                            }
+                            sh 'source deactivate'
+
+                            try{
+                                sh """
+                                    source activate ${condaPath}/envs/isis 
+                                    ctest -R _module_ -j4 -VV
+                                """
+                            }
+                            catch(e) {
+                                build_ok = false
+                                errors.add(env.STAGE_STATUS)
+                                println e.toString()
+                            }
+                            sh 'source deactivate'
+
+                            try{
+                                env.STAGE_STATUS = "Running gtests on ${label}"
+                                sh """
+                                    source activate ${condaPath}/envs/isis
+                                    ctest -R "." -E "(_app_|_unit_|_module_)" -j4 -VV
+                                """
+                            }
+                            catch(e) {
+                                build_ok = false
+                                errors.add(env.STAGE_STATUS)
+                                println e.toString()
+                            }
+                            sh 'source deactivate'
                         }
-                    }
-                    parallel builders
+                      }
+
+                      if(build_ok) {
+                          currentBuild.result = "SUCCESS"
+                      }
+                      else {
+                          currentBuild.result = "FAILURE"
+                          def comment = "Failed during:\n"
+                          errors.each {
+                              comment += "- ${it}\n"
+                          }
+                          setGitHubBuildStatus(comment)
+                      }
+                  }
+
+                  stage("Clean Up") {
+                    env.STAGE_STATUS = "Removing conda environment"
+                  }
```

Comment:
Can this stage go away? It doesn't do anything.

Reply:
I'll remove it.